### DISCUSSION TITLE
How to move to next line after auto-complete?

### DISCUSSION THREAD

**Discussion by ghost** [LazyVim/LazyVim#1921] [Q&A]
I apologize in advance if I've missed an obvious solution here. I'm using all the default LazyVim keybindings.

When using autocomplete, it will often add a character (e.g. a semicolon) to the end of the current line, after my current cursor position. Naturally, if I hit enter to move to the next line, the final character moves to the next line also. But if I try to prevent this by entering the character manually before hitting enter, it duplicates the character and the original still moves the next line on hitting enter.

As it is, if I want to use autocomplete and then continue typing a new line beneath, I need to leave insert mode and press 'o' to do so. But this messes with my flow. Surely there's a way to autocomplete the current line and move on to the next line without leaving insert mode? I've been searching Google and the documentation to no avail. Any help would be greatly appreciated!

**Comment by dpetka2001** [comment]
Lazyvim has a mapping `<C-CR>` to go to new line while in insert mode and completion.

**Reply by ayoblt** [reply]
I understand `<C-CR>` is Ctrl + ENTER right. Because this doesn't work for me
